gitbook-docs/security.md

```diff
@@ -80,25 +92,27 @@ If the login fails, the server will send a response like the following:
   "result": 401
 }
 ```
+The `result` codes are the same as normally used in HTTP.
 
 ### Providing Authorization to the Server in Subsequent Requests
 
 #### Web Based Clients
 
-Web based clients should be sure to include the cookie set in the authentication response in all subsequent requests.
-
-To logout, a web based client should send an HTTP PUT request to `/signalk/«version»/auth/logout`.
+Web based clients should be sure to include the cookie or `Authorization` HTTP header obtained from the authentication response in all subsequent requests.
 
 #### WebSockets Clients
 
 Clients can include the authentication cookie with the initial request.
 
 Clients can include the `Authorization` HTTP header with the initial connect request. The format of the header should
-be `{type} {token}`, for example `Authorization: JWT eyJhbGciOiJIUzI1NiIsI...ibtv41fOnJObT4RdOyZ_UI9is8`
+be `Bearer {token}`, for example `Authorization: Bearer eyJhbGciOiJIUzI1NiIsI...ibtv41fOnJObT4RdOyZ_UI9is8`
+
+Note that clients with a token can also include the `token` key in every signalk message sent over the websocket connection 
```

Comment:
Sorry about being a pest, but what does _can include a token_ exactly mean? A client _can_ include all kinds of data in the messages, but they won't have any effect unless specified here.

Reply:
Removed, so we can get this merged.